salt/states/boto_iam.py

```diff
@@ -498,13 +498,16 @@ def _case_password(ret, name, password, region=None, key=None, keyid=None, profi
     return ret
 
 
-def group_present(name, policies=None, policies_from_pillars=None, users=None, region=None, key=None, keyid=None, profile=None):
+def group_present(name, path='/', policies=None, policies_from_pillars=None, users=None, region=None, key=None, keyid=None, profile=None):
```

Comment:
Can you move this new kwarg to the end of the kwargs list to avoid an API change?